mlibrary/search
# Improve DatastorePage

# Overview
A lot of information is being loaded in `DatastorePage`. To help increase performance, variable assignments have been destructured, unused properties have been removed, and properties have been refined so it's only receiving the necessary information.

## Anything else?
`DatastoreMain` has been created and all related components and `useSelector` assignments have been moved to that, so they do not need to be loaded whenever `DatastorePage` is loaded.

## Testing
- Make sure the PR is consistent in these browsers:
- Run accessibility tests:
- Check through the site to see if anything is broken.
